examples/batch.md

```diff
@@ -0,0 +1,29 @@
+### Running batch tasks in xpk
+
+### Preparation
+
+This section should describe how to prepare volumes and add external files executed in script.
+
+### Execution
+
+To run xpk batch script run:
+```bash
+python3 xpk.py batch slurm_sample.sh
+```
+
+You may pass script arguments to executed script by running:
+```bash
+python3 xpk.py batch slurm_sample.sh arg1 arg2
```

Comment:
It might be helpful for beginner users to add here example --cluster, --zone and --project (I think these 3 are required, correct me if wrong). I had to think about it for a minute, so this info will be beneficial when trying the command first time.

Reply:
only --cluster is required, but will add it